build/css/Forum.css

```diff
@@ -0,0 +1,36 @@
+.input[type=text] {
```

Comment:
I would be careful with css like this, it applies globally to all elements referencing that stylesheet.